src/main/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoader.java

```diff
@@ -60,6 +62,13 @@ protected void init() {
                     activeVenueDetailsMapByEpimsId.put(line[15], venueDetails);
                 }
             });
+            activeVenueEpimsIdsMapByRpc.putAll(
+                    activeVenueDetailsMapByEpimsId // use active otherwise closed epimsIds (000000) picked up
+                    .values()
+                    .stream()
+                    .collect(Collectors.groupingBy(VenueDetails::getRegionalProcessingCentre,
+                            Collectors.mapping(VenueDetails::getEpimsId, Collectors.toList()))));
```

Comment:
Might be worth keeping the full venue details for future use

Reply:
Okay, I should be able to refactor to retain a list of VenueDetails as values for map instead of List of epimsId strings, this will change the tests, etc...